katas/main/src/pokerhands/jorge/3.List.cs

```diff
@@ -0,0 +1,68 @@
+﻿namespace PokerGame.HandNumsGrouped.Jorge;
+using System.Linq;
+using PokerGame.Dictionary.Jorge;
+using PokerGame.Score.Jorge;
+
+public class PokerGameOrderList
+{
+    public List<string> pairArray = new List<string>(); // List to know pairs.
+
+    internal string[] ListGroup(List<string> currentPokerHand)
```

Comment:
This method is doing too much, it would be a good idea to break it in smaller pieces